.github/CONTRIBUTING.md

```diff
@@ -98,10 +99,9 @@ $ make html
 
 The built documentation can then be found in `docs/_build/html/`.
 
-To file a pull request, create a new branch on top of the upstream repository:
+To file a pull request, create a new branch on top of the upstream repositorie's `main` branch:
 
 ```console
-$ git fetch --all
```

Comment:
I think it's feasible that days go by since that call, because we kinda suggest to start hacking and only _then_ start branch?

Reply:
Sounds reasonable. But do we need to call `git fetch upstream` explicitly then?

Reply:
You mean as in "replacing --all by upstream" or as in "removing the earlier fetch"?

The former works for me (you could also add a comment `$ git fetch upstream  # ensure upstream/main hasn't changed in the meantime`)?

Reply:
> "removing the earlier fetch"?

Yes, since `--all` fetches from all remotes, it will also fetch the tags from `upstream`?

Reply:
What would the work be based on then? Is the assumption that it’s new enough due to the clone? ISTM that it’s better to have an extra fetch than a bad start?

Reply:
> What would the work be based on then?

Maybe I got something wrong, but isn't `git fetch <remote>` meaningless if `git fetch --all` is called afterwards? 

But I'm also fine with keeping both fetches (and add your comment).

Reply:
> Maybe I'm missing something, but to me it looks like the first fetch is for having an up to date repo when starting to work and the second one is to based the branch on the latest changes that might have arrived while working?

But doesn't working start **after** the branch has been created? Both fetches are suggested before that. I think the first fetch _can_ safely be removed.

> It would probably make sense to change to `--all` to `upstream` to keep it consistent?

I think related to the comment I left in the other discussion, this would make sense.

Reply:
> > "removing the earlier fetch"?
>
> Yes, since --all fetches from all remotes, it will also fetch the tags from upstream?

This seems to imply a connection between the two so I'd like to point that there's no direct relation between `--all` and the tags. The tags are fetched based on how the remote is set up in the config for each specific remote or an explicit CLI request, not based on whether all the remotes are fetched or just one.

Reply:
Yes, that is why I doubted the earlier `fetch` was needed.